### ISSUE TITLE
Make update check optional

### ISSUE THREAD

**Issue by jbergstroem** [npm/npm#17150]
#### I'm opening this issue because:

  - [x] npm is doing something I don't understand.

#### What's going wrong?

It'd be great if I could get to choose whether or not npm would check for updates.

(sorry if dup, spent 10 minutes looking for open and closed issues)

#### How can the CLI team reproduce the problem?

use `npm install` with a npm version older than $current.

### supporting information:

 - `npm -v` prints: npm 5.0.0
 - `node -v` prints: node.js v.8.0.0

**Comment by oriash93** [comment]
Hey,

I tried to reproduce this, but I think I missed something.
I use npm 5.0.0 (current version is 5.6.0).
When I run ```npm install``` it won't update my npm version.

Is this what you meant?

**Comment by jbergstroem** [comment]
@oriash93 yes, I've observed that `npm` (occasionally) self-checks for updates.
